examples/life-expectancy/README.md

```diff
@@ -0,0 +1,219 @@
+# Life Expectancy
+This example provides the complete code to perform the following tasks:
+- Data Ingestion from CSV
+- Data Processing
+- Data Publication in form of Visual Report/Dashboard
```

Comment:
I might be wrong but I do not see the code for the visual report/ dashboard. I could see the scenario ending with merging the two demos but no usage of these datasets afterwards.